architecture/README.md

```diff
@@ -33,76 +30,82 @@ types of roles. The TiDB community has the following organizations:
 
-[TiDB Contributors](./contributor-list.md)
+See [TiDB Contributors](./contributor-list.md)
 
 ### Active Contributor
 
-TBD
+Active Contributors are continuously active contributors in the community. They can have issues and PRs assigned to them, participate in SIGs through GitHub teams, and pre-submit tests are automatically run for their PRs. See `member-list.md` under the specific SIG for a list of Active Contributors.
+
+#### How to become an Active Contributor
+
+- Contributed 8 PRs and above to a specific SIG within one year
+- Sponsored by two incumbent Reviewers in a SIG
+
+#### Responsibilities and privileges
+
+- Get the profile listed on the membership page of the corresponding SIG
 
 ### Reviewer
 
-Reviewers are able to review code for quality and correctness on some part of a
-subproject. They are knowledgeable about both the codebase and software
-engineering principles.
+Reviewers are responsible for reviewing the code submitted by the community to ensure the quality and correctness; generated out of Active Contributors in a specific SIG.
+
+Reviewer status is scoped to the part of the codebase under the SIG.
+See `member-list.md` under the specific SIG for a list of Reviewers of the SIG.
 
-#### How to become a Reviewer?
+#### How to become a Reviewer
 
-* Knowledgeable about the codebase
-* Sponsored by 2 Committers
-* Merged at least 20 PRs 
+- Familiar with the codebase
+- Demonstrated a recoganized level of community engagement in the SIG development, via contributions that have met one or more of the minimal criteria as defined below：
+  - Merged 20+ PRs (may vary among SIGs)
+  - Handled 1+ more issue of Medium difficulty or higher
+- Sponsored by at 2 Committers of the SIG
 
 #### Responsibilities and Privileges
 
-* Responsible for project quality control
-    * Focus on code quality and correctness, including testing and factoring 
-    * May also review for more holistic issues
-* Expected to be responsive to review PR on time
-* Assigned test and fix bugs in subproject 
-* Have the rights to approve PR
-* Participate in the future seminar
+- Ensure the correctness of the code submitted to the SIG
+- Respond promptly to PRs
+- Participate in SIG meetings regularly
+- Approve code reviews with LGTM (Looks Good To Me)
+- Get the profile listed on the membership page of the corresponding SIG
 
 ### Committer
 
-Committers are developers of the projects and are elected from contributors.
-Currently.
+Committers come from Reviewers. Committer is scoped to the specific SIG where the promotion applies. Committers are recoganized by Maintainers or SIG Tech Leads for their outstanding contributions to the specific SIG.
 
-#### How to become a Committer?
+Committer status is scoped to the part of the codebase under the SIG. See `member-list.md` under the specific SIG for a list of Committers of the SIG.
+
+#### How to become a Committer
 
 A new Committer must be nominated by any existing Maintainer or PMC member, and
```

Comment:
Committer should be sponsored by 2 maintainers, but not necessarily voted by PMC